DEV-3/DEV-3/Converter.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace DEV_3
+{
+    /// <summary> 
+    /// Converter accept from the command line two arguments
+    /// and convert number (1 argument) in the new numeral system (2 argument)
+    /// </summary> 
+    class Converter
+    {       
+        static void Main(string[] args)
+        {
+            try
+            {
+                int number = int.Parse(args[0]);
+                int newBaseOfNumberSystem = int.Parse(args[1]);                
+                if (newBaseOfNumberSystem < 2 || newBaseOfNumberSystem > 20)
+                {
+                    throw new ArgumentOutOfRangeException("wrong base of new numeral system.");
+                }                
+                NumeralSystemConverter EntryPointObject = new NumeralSystemConverter();
```

Comment:
Better to pass number and new system radix to class via construstor, not as method arguments